### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lburgazzoli**
- update dependencies
- reduce testcontainers llog verbosity

### CODE REVIEW

File: jetcd-core/src/test/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <Logger name="org.testcontainers" level="INFO" additivity="true"/>
+    <Logger name="🐳 [gcr.io/etcd-development/etcd:v3.3]" level="WARN" additivity="true"/>
+
+    <!-- main logger -->
+    <Root level="INFO">
+      <AppenderRef ref="STDOUT" />
+    </Root>
+  </Loggers>
+
+</Configuration>
```
[the remark this answers is not in the document]
Comment (lburgazzoli) [reply jetcd-core/src/test/resources/log4j2.xml]: can someone add a checkstyle rule for this ?

**Comment by lburgazzoli** [comment]
@xiang90 fixed

File: jetcd-core/src/test/resources/log4j2.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fanminshi) [reply jetcd-core/src/test/resources/log4j2.xml]: @lburgazzoli file an issue to track that?

[the remark this answers is not in the document]
Comment (lburgazzoli) [reply jetcd-core/src/test/resources/log4j2.xml]: already implemented :)

**Pull request merged**